proto/rill/admin/v1/api.proto

```diff
@@ -45,6 +45,24 @@ service AdminService {
     };
   }
 
+  // UpdateOrganizationBillingPlan updates the billing plan for the organization
+  rpc UpdateOrganizationBillingPlan(UpdateOrganizationBillingPlanRequest) returns (UpdateOrganizationBillingPlanResponse) {
+    option (google.api.http) = {
+      patch: "/v1/organizations/{org_name}/billing/plan",
+      body: "*"
+    };
+  }
```

Comment:
Should we rename this to `UpdateOrganizationBillingSubscription` to align with the naming of `GetOrganizationBillingSubscription`?